app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -107,6 +107,51 @@ def pvm_instances
         :value        => instance.proc_type,
         :read_only    => true
       )
+
+      # saving pin_policy
+      persister.vms_and_templates_advanced_settings.build(
+        :resource     => ps_vmi,
+        :name         => 'pin_policy',
```

Comment:
We should go ahead and add some quick checks to our refresher spec:
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb

I don't think you even need to record a new VCR cassette:
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/spec/vcr_cassettes/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher.yml#L585